lib/parser_multiple.rb

```diff
@@ -33,12 +34,14 @@ def word_count(headline)
     headline.first.split(' ').length
   end
 
-  def number_first_position(headline)
-    headline.first.split(' ')[0].to_i != 0 ? 1 : 0
-  end
-
   def flagged_pronoun_first_position(headline)
-    starting_pronouns.inject(0) { |r, w| w == headline.first.split(' ')[0] ? r + 1 : r }
+    starting_pronouns.inject(0) { |r, w| headline.first.split(" ")[0].to_i != 0 && r == 0 || headline.first.split(' ')[0] == w ? r + 1 : r }
```

Comment:
Maybe change the name to first_word as it checks numbers or pronouns in first position